graph/model/discussion.go

```diff
@@ -38,6 +38,21 @@ type Discussion struct {
 	IdleMinutes           int                          `json:"idle_minutes"`
 	IconURL               *string                      `json:"icon_url"`
 	DiscussionJoinability DiscussionJoinabilitySetting `json:"discussion_joinability"`
+	LastPostID            string                       `json:"last_post_id"`
+	LastPostCreatedAt     time.Time                    `json:"last_post_created_at"`
+}
+
+type DiscussionInput struct {
```

Comment:
moving here to add the LastPost fields. this allows us to use the same struct without needing to expose the fields to gql